File: 백준/Bronze/2703. Cryptoquote/Cryptoquote.c
#include <stdio.h>

int main(void)
{
	int t;
	char cipher[101];
	char rule[27];
	scanf("%d", &t);
	
	for (int i = 0; i < t; i++)
	{
		scanf(" %[^\n]s", cipher);
        scanf("%s", rule);
		for (int j = 0; cipher[j]; j++)
		{
			if (cipher[j] == ' ')
			{
				fputc(' ', stdout);
			} else {
    			fputc(rule[cipher[j] - 65], stdout);
            }
		}
		fputc('\n', stdout);
	}
	return 0;
}

File: 백준/Bronze/2703. Cryptoquote/Cryptoquote.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    T = int(input())
    for _ in range(T):
        cipher = input()
        rule = input()
        dap = ""
        for i in range(len(cipher)):
            if cipher[i] == " ":
                dap += " "
            else:
                dap += rule[ord(cipher[i])-65]
        print(dap)


if __name__ == '__main__':
    main()